<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pointage Periscolaire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Présence en cours:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Présence terminée:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminer toutes les présences en cours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vérifier le fichier de pointage</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listIN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listOUT">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>POINTAGE PERISCOLAIRE</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>961</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>OUTILS</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="selGroup">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>191</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>    EDITER LA SELECTION</string>
    </property>
    <widget class="QLabel" name="selStatut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelIN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Arrivée</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>511</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="selAdh">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>35</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelOUT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Départ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Statut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelIN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer du pointage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>36</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Adhérent</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="selIN">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelOUT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer le départ</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="selOUT">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selOK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider Modifications</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler Modifications</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pointOUT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>161</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pointer le départ</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>511</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>511</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>19</y>
       <width>191</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FCC</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>selStatut</zorder>
    <zorder>labelIN</zorder>
    <zorder>line_3</zorder>
    <zorder>selAdh</zorder>
    <zorder>labelOUT</zorder>
    <zorder>label_13</zorder>
    <zorder>cancelIN</zorder>
    <zorder>label_12</zorder>
    <zorder>selIN</zorder>
    <zorder>cancelOUT</zorder>
    <zorder>selOUT</zorder>
    <zorder>selOK</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pointOUT</zorder>
    <zorder>line_6</zorder>
    <zorder>line_7</zorder>
    <zorder>line_8</zorder>
   </widget>
   <widget class="QFrame" name="selHider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>741</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="info">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Ecole</string>
     </property>
    </widget>
    <widget class="QLabel" name="Ecole">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;padding: 2px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid</string>
     </property>
     <property name="text">
      <string>Heure</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="messageTitle">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro d'adhérent:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Pointer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LISTING">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Listing</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancelALL">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler tout le pointage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QUIT">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>QUITTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MORE">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>listIN</zorder>
   <zorder>listOUT</zorder>
   <zorder>label_6</zorder>
   <zorder>line_4</zorder>
   <zorder>label_15</zorder>
   <zorder>selGroup</zorder>
   <zorder>info</zorder>
   <zorder>frame</zorder>
   <zorder>cancelALL</zorder>
   <zorder>selHider</zorder>
   <zorder>QUIT</zorder>
   <zorder>MORE</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
